### ISSUE TITLE
hh_client error in ClassConsumer.php

### ISSUE THREAD

**Issue by a1exsnow** [fredemmott/definition-finder#34]
Hi!

Can you please check this error:

```
vendor/fredemmott/definition-finder/src/consumers/ClassConsumer.php:32:9,49: This equality test has incompatible types (Typing[4157])
  vendor/fredemmott/definition-finder/src/consumers/ClassConsumer.php:13:13,31: This is a value of enum FredEmmott\DefinitionFinder\ClassDefinitionType
  vendor/fredemmott/definition-finder/src/definitions/DefinitionType.php:5:22,24: This is a value of enum FredEmmott\DefinitionFinder\DefinitionType
```

version: v1.3.3

`hhvm --version
HipHop VM 3.16.0-dev (rel)
`

**Comment by fredemmott** [comment]
Please double-check that you actually have v1.3.3 installed, not just in your composer.lock files - that was fixed in the latest commit in v1.3.3: https://github.com/fredemmott/definition-finder/commit/8c89d5218cf4e4b447fa9132be51b4f75f16b1d7

**Comment by fredemmott** [comment]
![screen shot 2016-11-07 at 9 05 01 am](https://cloud.githubusercontent.com/assets/360927/20067256/483833be-a4c9-11e6-842f-0bbe0f662b29.png)

^^^^ that as me, reporting to github :/


Closing as this isn't reproducible and there was a specific fix for it already.
